index.html

```diff
@@ -15,18 +15,28 @@
           <a class="logo" href="#">Abel Tsegaye</a>
         </logo>
         <img class="mobileMenu" src="images\IconMenu.png" alt="Menu" />
+        <ul class="desktopMenu">
+          <li class="desktopLink">Portfolio</li>
+          <li class="desktopLink">About</li>
+          <li class="desktopLink">Contact</li>
+          <li class="desktopLink">
+            <img class="mail" src="/images/Mail.svg" alt="Mail" />
+          </li>
+        </ul>
       </nav>
```

Comment:
Please consider removing the property of `position: fixed;` from your nav as according to the design it does not to be fixed.